modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -64,13 +55,14 @@ func (suite *KeeperTestSuite) TestEscrowPacketFee() {
 
 		suite.Run(tc.name, func() {
 			suite.SetupTest() // reset
+			suite.coordinator.Setup(suite.path)
 
 			// setup
 			refundAcc = suite.chainA.SenderAccount.GetAddress()
 			ackFee = validCoins
 			receiveFee = validCoins2
 			timeoutFee = validCoins3
-			packetId := &channeltypes.PacketId{ChannelId: validChannelId, PortId: types.PortID, Sequence: uint64(1)}
+			packetId := &channeltypes.PacketId{ChannelId: suite.path.EndpointA.ChannelID, PortId: transfertypes.PortID, Sequence: uint64(1)}
```

Comment:
Removed `types.PortID` since it was a holdover from earlier design. Now, the fee module does not own or bind to any ports. It is responsibility of an underlying app like transfer to bind to unique port

Reply:
We should probs change this elsewhere also

Reply:
I removed it completely so it won't be anywhere else